src/mixins/regions.js

```diff
@@ -123,6 +123,9 @@ export default {
   // Accepts the region name
   // getRegion('main')
   getRegion(name) {
+    if (!this.isRendered()) {
```

Comment:
I think we need to use `_isRendered` instead of the method to support non-Marionette Views.